app/controllers/admin/users_controller.rb

```diff
@@ -53,7 +54,23 @@ def update
       # TODO: [@rhymes] in the new Admin Member view this logic has been moved
       # to Admin::Users::Tools::CreditsController and Admin::Users::Tools::NotesController#create.
       # It can eventually be removed when we transition away from the old Admin UI
-      Credits::Manage.call(@user, user_params)
+      credit_params = nil
+
+      if FeatureFlag.enabled?(:admin_member_view)
+        credit_params = {}
+
+        if user_params[:credit_action] == "Add"
+          credit_params[:add_credits] = user_params[:credit_amount]
+        end
+
+        if user_params[:credit_action] == "Remove"
+          credit_params[:remove_credits] = user_params[:credit_amount]
+        end
+      else
+        credit_params = user_params
+      end
+
+      Credits::Manage.call(@user, credit_params)
```

Comment:
This is just to get something working. I'm sure there's a more "Rails"y way of doing this. Also, this is some logic I'm doing in the controller, so I imagine we'd want a helper/utility to do this. Maybe this is also the time to do the TODO that Rhymes mentioned.